api/repositories/pod_repository.go

```diff
@@ -19,6 +19,7 @@ const (
 	workloadsContainerName = "opi"
 	cfInstanceIndexKey     = "CF_INSTANCE_INDEX"
 	eiriniLabelVersionKey  = "workloads.cloudfoundry.org/version"
+	cfProcessGuidKey       = "workloads.cloudfoundry.org/guid"
```

Comment:
Birdrock created a chore to improve this label key name:
https://github.com/cloudfoundry/cf-k8s-controllers/issues/410